### ISSUE TITLE
Uses 100% CPU simultaneously with GPU

### ISSUE THREAD

**Issue by marshalleq** [nagadit/DeepFaceLab_Linux#76]
Has anyone else found this?  I don't recall this happening in the windows version.

Thanks.

**Comment by gteachey** [comment]
I got nothing for this one.....my CPU is 91% idle while my GPU is chugging away training at the moment.

**Comment by marshalleq** [comment]
Yeah crazy right? I know there is that question in the startup script, that says how many CPU cores to use - which I usually max out - I always figured it was only for tasks that GPU doesn't do, but maybe it doesn't work like that?  Maybe it was windows that was broken.

**Comment by marshalleq** [comment]
OK, I've confirmed that this directly changes when selecting how many CPU cores you want to use in the startup script.  While that might sound obvious, note that I have not selected to use CPU for training,  only GPU.  So it would seem to me that something in the script is enabling the CPU for training without user input - unless there is some other CPU usage that is bugged.  I can see clearly in top that the cpu is in use by python3.9, but that's about it so far.

**Comment by tailangjun** [comment]
I have the same problem on Ubuntu22.04, my graphics card RTX4090 only uses 396MB of video memory
